File: 2ª Unidade/Normalização/Curso.pl
% Relação Curso

curso( [idCurso, cNome, cDescricao, cargaHoraria]  ).

fds(  [ [[idCurso], [ cNome, cDescricao, cargaHoraria]] ]  ).

% Query: curso(R), fds(F), threenf(R,F,T).

% Resultado: R == T
% F = [[[idCurso], [cNome, cDescricao, cargaHoraria]]],
% R = [idCurso, cNome, cDescricao, cargaHoraria],
% T = [[idCurso, cDescricao, cNome, cargaHoraria]]

File: Desenvolvimento/src/entidades/Curso.java
package entidades;

public class Curso {
	
	private int idCurso;
	private String cNome;
	private String cDescricao;
	private int cargaHoraria;
	
	public Curso(int idCurso, String cNome, String cDescricao, int cargaHoraria) {
		super();
		this.idCurso = idCurso;
		this.cNome = cNome;
		this.cDescricao = cDescricao;
		this.cargaHoraria = cargaHoraria;
	}

	public int getIdCurso() {
		return idCurso;
	}

	public void setIdCurso(int idCurso) {
		this.idCurso = idCurso;
	}

	public String getcNome() {
		return cNome;
	}

	public void setcNome(String cNome) {
		this.cNome = cNome;
	}

	public String getcDescricao() {
		return cDescricao;
	}

	public void setcDescricao(String cDescricao) {
		this.cDescricao = cDescricao;
	}

	public int getCargaHoraria() {
		return cargaHoraria;
	}

	public void setCargaHoraria(int cargaHoraria) {
		this.cargaHoraria = cargaHoraria;
	}

}
